topology/probes/k8s/container.go

```diff
@@ -126,17 +151,17 @@ func newDockerIndexer(g *graph.Graph) *graph.MetadataIndexer {
 	return graph.NewMetadataIndexer(g, g, m, dockerPodNamespaceField, dockerPodNameField, dockerContainerNameField)
 }
 
-func newContainerLinker(g *graph.Graph, subprobes map[string]Subprobe) probe.Probe {
-	podProbe := subprobes["pod"]
-	if podProbe == nil {
+func newContainerDockerLinker(g *graph.Graph, subprobes map[string]Subprobe) probe.Probe {
```

Comment:
Almost all of the changes in this function is about renaming things. Please stop renaming local variables

Reply:
this is not about renaming its' about splitting functionality and enabling re-use of the listener to k8s client events (for containers) with out taking the other code (which assuming a 1-1 mapping between a k8s resource and a graph node)